### ISSUE TITLE
The button to import is missing

### ISSUE THREAD

**Issue by eibarra466996** [grokability/snipe-it#13053]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

when you want to import anything, the button does not appear

### Reproduction steps

1.enter the import button
2.select CSV file
3.The button to upload or import does not appear
...

### Expected behavior

I should be able to import the assets

### Screenshots

![image](https://github.com/snipe/snipe-it/assets/131706912/523f1bf9-9bbc-477f-840a-4b10330bdb64)

### Snipe-IT Version

Version v6.1.0 - build 10161 (master)

### Operating System

Windows 10

### Web Server

Apache

### PHP Version

5.2.0

### Operating System

Windows

### Browser

Chrome\Edge

**Comment by Godmartinz** [comment]
Have you followed the proper naming conventions in your csv as per the [documentation](https://snipe-it.readme.io/docs/importing) I am not able to reproduce your issue. 
<img width="866" alt="image" src="https://github.com/snipe/snipe-it/assets/47435081/d979f2b1-15f0-4466-8e77-98378deab1ca">

**Comment by eibarra466996** [comment]
@Godmartinz Yes, I am trying with the example document and the option to import is not enabled either, I updated the version a week ago, could something go wrong? Is there something I need to check in the configuration?

![image](https://github.com/snipe/snipe-it/assets/131706912/9166ad43-ebe3-4092-8247-5c28c0efbadd)

**Comment by snipe** [comment]
Can you check your browser console? It looks like some of the CSS or Javascript might not be loading.

**Comment by snipe** [comment]
Also I don't think you're actually running PHPv5.2.0 - Snipe-IT v6 doesn't run on PHP 5.

**Comment by eibarra466996** [comment]
@snipe Could it be a problem with the version?


![image](https://github.com/snipe/snipe-it/assets/131706912/bfe29fb3-5ec3-4f37-aa73-e69510ec74c4)

**Comment by snipe** [comment]
I think that's just your version of phpMyAdmin, not PHP itself. If you go to /admin in Snipe-IT it should show you the version of PHP you're running.

**Comment by eibarra466996** [comment]
@snipe sure is 8.1.6

![image](https://github.com/snipe/snipe-it/assets/131706912/9958aacb-5d93-466e-b735-7bc072250537)

**Comment by snipe** [comment]
Okay, that's more like it - what's in your browser console?

**Comment by snipe** [comment]
Hi there - We haven't heard back in a bit, so I'm going to close this ticket for now, but will re-open it if you're still having issues.

![hackerman-hacking](https://user-images.githubusercontent.com/197404/31936003-27a7f6d8-b865-11e7-8f5e-eb03062925f4.gif)
